app/models/spree/gateway/amazon.rb

```diff
@@ -105,7 +105,7 @@ def capture(amount, amazon_checkout, gateway_options={})
       t = order.amazon_transaction
       t.capture_id = response.fetch("CaptureResponse", {}).fetch("CaptureResult", {}).fetch("CaptureDetails", {}).fetch("AmazonCaptureId", nil)
       t.save!
-      return ActiveMerchant::Billing::Response.new(response.fetch("CaptureResponse", {}).fetch("CaptureResult", {}).fetch("CaptureDetails", {}).fetch("CaptureStatus", {})["State"] == "Completed", "OK", response)
+      return ActiveMerchant::Billing::Response.new(response.fetch("CaptureResponse", {}).fetch("CaptureResult", {}).fetch("CaptureDetails", {}).fetch("CaptureStatus", {})["State"] == "Completed", "OK", response, {authorization: t.capture_id})
```

Comment:
This is great. Could we add a spec somewhere that verifies that `spree_payments.response_code` does get set after performing a capture?